woocommerce/woocommerce-gutenberg-products-block
# Product Gallery Thumbnails: Only output thumbnails markup when multiple product thumbnails are present

This PR prevents an empty `Thumbnails` markup from being outputted on the page.

## Why

When testing the following PR: https://github.com/woocommerce/woocommerce-blocks/pull/10743#pullrequestreview-1598271926 I have noticed that the Product Gallery Thumbnails block outputs empty markup when there is only one product image.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. With a block theme, open `Appearance > Editor`.
2. Open `Templates > Single Product`.
3. Select `Convert to Blocks` if you are using the Classic Templates.
4. Add the `Product Gallery` block.
5. Ensure the `Thumbnails` are enabled.
6. Open a product page on the frontend with multiple product images and verify the thumbnails are displaying correctly.
7. Open a product page on the frontend with just one product image, inspect the code and verify there is no empty `Thumbnails` block markup.



## Screenshots or screencast



| Before | After |
| ------ | ----- |
|<img width="1008" alt="Cursor_i_T-Shirt_with_Logo_–_newproductgallery" src="https://github.com/woocommerce/woocommerce-blocks/assets/905781/d508e652-d75f-4d2d-ba0b-e86d70200d82">|<img width="1134" alt="Cursor_i_T-Shirt_with_Logo_–_newproductgallery-2" src="https://github.com/woocommerce/woocommerce-blocks/assets/905781/6312274c-7f5e-4e37-80c8-d463230db0fd">|

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Add suggested changelog entry here.
